connPFM/connectivity/connectivity_utils.py

```diff
@@ -136,35 +138,21 @@ def calculate_hist_threshold(hist, bins, percentile=95):
     return thr
 
 
-def surrogates_histogram(
-    surrprefix,
-    sursufix,
-    masker,
-    hist_range,
-    numrand=100,
-    nbins=500,
-    percentile=95,
+def sum_histograms(
+    hist_list,
+    numrand,
```

Comment:
You could avoid using this variable by extracting the number of surrogates as `len(hist_list)` and probably the same with `nbins`

Reply:
Done in [1526086](https://github.com/SPiN-Lab/connPFM/pull/63/commits/1526086137cdcc8e2db840a17e52e28670daa4c5).